logstash-output-elasticsearch.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'ftw', '~> 0.0.42'
   s.add_development_dependency 'addressable', "~> 2.3.0" # used by FTW. V 2.5.0 is ruby 2.0 only.
   s.add_development_dependency 'logstash-codec-plain'
+  s.add_development_dependency 'json' # used by spec/unit/outputs/elasticsearch/http_client/pool_spec.rb
```

Comment:
I ... am not sure. It's clearly referenced here: https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/master/spec/unit/outputs/elasticsearch/http_client/pool_spec.rb#L3

Maybe everyone who already works on the project happens to have it on their classpath for other reasons? I figured I ran into it because I started from a blank-slate environment.